### ISSUE TITLE
TF warning

### ISSUE THREAD

**Issue by clemens-tolboom** [clemens-tolboom/tf-mnist#2]
While doing #1 TensorFlow gets updated but it's keras lib reports a warning.

> WARNING:tensorflow:From /Users/clemens/.local/share/virtualenvs/tf-mnist-0POXUKpW/lib/python3.6/site-packages/tensorflow/python/ops/resource_variable_ops.py:435: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
Instructions for updating:
Colocations handled automatically by placer.
WARNING:tensorflow:From /Users/clemens/.local/share/virtualenvs/tf-mnist-0POXUKpW/lib/python3.6/site-packages/tensorflow/python/keras/layers/core.py:143: calling dropout (from tensorflow.python.ops.nn_ops) with keep_prob is deprecated and will be removed in a future version.
Instructions for updating:
Please use `rate` instead of `keep_prob`. Rate should be set to `rate = 1 - keep_prob`.
